docs/source/example_seq2seq.rst

```diff
@@ -1,12 +0,0 @@
-..
-  Copyright (c) Facebook, Inc. and its affiliates.
-  This source code is licensed under the MIT license found in the
-  LICENSE file in the root directory of this source tree.
-
-agents.example_seq2seq
-======================
-
-Reference implementation for a simplified Seq2Seq model.
-
-.. automodule:: parlai.agents.examples.seq2seq
```

Comment:
Will this auto generate contents?

Reply:
Yes https://parl.ai/docs/agent_refs/examples.html